src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -4,13 +4,21 @@
 
 use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
+use Illuminate\Events\Dispatcher;
 use Illuminate\Support\Collection;
 use Illuminate\Console\OutputStyle;
 use Illuminate\Filesystem\Filesystem;
 use Illuminate\Database\ConnectionResolverInterface as Resolver;
 
 class Migrator
 {
+    /**
+     * The event dispatcher instance.
+     *
+     * @var \Illuminate\Contracts\Events\Dispatcher
+     */
+    protected static $dispatcher;
```

Comment:
The static keyword isn't needed here. Also, renaming this to `$events` will make it clearer what it is.
```suggestion
    protected $events;
```

Reply:
Done